src/python/bot/fuzzers/builtin.py

```diff
@@ -86,6 +91,8 @@ def run(self, input_directory, output_directory, no_of_files):
       from bot.untrusted_runner import file_host
       corpus_directory = file_host.rebase_to_worker_root(corpus_directory)
 
+    self.get_mutator_plugin(fuzzer_binary_name)
```

Comment:
This should go into the launcher. Otherwise this may cause complications with supporting OSS-Fuzz since this code runs on the host, while the launcher runs on the worker.

Reply:
Done.